addons/calendar/models/calendar_event.py

```diff
@@ -1081,6 +1081,16 @@ def _sync_activities(self, fields):
     def _get_trigger_alarm_types(self):
         return ['email']
 
+    def _process_event_alarms(self, events_by_alarm):
```

Comment:
Maybe we could be a bit more precise actually ?
```suggestion
    def _setup_event_recurrent_alarms(self, events_by_alarm):
```